src/api-engine/api/config.py

```diff
@@ -4,6 +4,8 @@
 CELLO_HOME = "/opt/cello"
 FABRIC_TOOL = "/opt/bin"
 FABRIC_CFG = "/opt/node"
+FABRIC_NODE = "/opt/fabric"
```

Comment:
No code is using the two new variables?

FABRIC_NODE = "/opt/fabric"
PRODUCTION_NODE = "/opt/production"

Reply:
We will use these variables to find local mapping volume paths for HLF node containers, and then remove them after calling delete node API endpoint.